### ISSUE TITLE
Another request/crafting pipe bug

### ISSUE THREAD

**Issue by Flow86** [RS485/LogisticsPipes#132]
I've set up a 2 crafting pipes which produce the same item, but have different input.

example:

copper ore: 1x (= 1x copper)
copper dust: 9x (= 1x copper)

requesting copper it will use the "copper-ore" crafting pipe producing one copper.
requesting another copper it says "1x copper ore missing" - but shouldnt it try "all" crafting pipes providing "copper"?

**Comment by ArtForz** [comment]
Can't reproduce the request actually failing, but something with the used/missing is off when there's not enough materials to do it.

**Comment by Flow86** [comment]
well it only happens via CC. via request pipe itself it works always - dont know whats different there?

**Comment by bziemons** [comment]
I finally came around to actually test this with the current build, and I do not have this problem. I had a simple setup with a redstone furnace that takes one pulverized copper and then a pulverizer and a furnace taking one ore. The results were 1x copper ingot in both cases, so the furnace with the pulverizer always left one ingot behind. When there was just one ore and one pulverized copper in the system, I requested one ingot twice and it worked.
